components/storage/blockdevice/COMPONENT_SPIF/SPIFBlockDevice.cpp

```diff
@@ -169,7 +170,7 @@ int SPIFBlockDevice::init()
             // SST devices come preset with block protection
             // enabled for some regions, issue write disable instruction to clear
```

Comment:
Should be changed to "issue global protection unlock to clear"

Reply:
@offirko Done 😃 
But what do you mean by:

> Should be changed to "issue global protection unlock to clear"

Reply:
Furthermore I added the command to SPIFReducedBlockDevice.cpp, it was there but only as a "magic number" and not as a command.